charts/opendj/templates/stateful-set.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: opendj
+  labels:
+    app: opendj
+    {{- template "opendj.labels" }}
+spec:
+  serviceName: opendj
+  replicas: 1
+  selector:
+    matchLabels:
+      app: opendj
+  template:
+    metadata:
+      labels:
+        app: opendj
+    spec:
+      serviceAccountName: opendj-sa
+      securityContext:
+        # The OpenDJ process runs as a non-root user (1001) in its container. We need to make sure
+        # that the mounted volume is writeable for that user.
+        fsGroup: 1001
+      containers:
+      - name: opendj
+        image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 1389
+          protocol: TCP
+        - containerPort: 1636
+          protocol: TCP
+        - containerPort: 4444
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        env:
+        - name: BASE_DN
+          # This value should be replaced by environment.
+          value: "dc=dsde-{{ .Values.global.terraEnv }},dc=broadinstitute,dc=org"
```

Comment:
So this gets defined in another values file? Is there any way to trace that file to the fact that opendj uses it here? Feels a little buried and odd partially having its own values file and partially relying on the terra global values.

I think we might even want to make the whole base domain value a template value since the whole string is necessary, it's just happens to be the case only the env part is modified.

Reply:
Makes sense, I'll make it a values file parameter